tests/udp_socket.rs

```diff
@@ -387,7 +391,15 @@ fn unconnected_udp_socket_connected_methods() {
     );
 
     // Socket is unconnected, but we're using an connected method.
-    assert_error(socket1.send(DATA1), "address required");
+    if cfg!(not(target_os = "windows")) {
```

Comment:
Same here as above.

Reply:
Not sure if possible, there are different error messages on Windows vs other platforms. How would you have one assert checking two different expected messages?

Reply:
I don't think mio makes any guarantees as to error messages. It doesn't matter terribly to me if the test **only** check that an error is received **or** it adds cfg flags to test messages per platform.

Reply:
You can just change this to `assert_error(socket1.send(DATA1), "address")` and add a comment saying it must say something about missing an address. Then we don't have this complex `cfg!` stuff.

Reply:
I'd rather not change this, unless you have a strong reason against it. Having cfg! in tests doesn't feel complex at all, actually it's weird we don't have more cfg! in them. Having just an assert_error(socket1.send(DATA1), "address") doesn't say much about the actual error you are expecting there. "address required" is enough to get an idea,  "no address was supplied. (os error 10057)" is even better. IMO tests should be as explicit as possible.